rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/ApplicationContextImpl.java

```diff
@@ -71,15 +70,15 @@ public void setRestServerUrl(String restServerUrl) {
 
     @Override
     public SchedulerRestClient getRestClient() {
-        HttpClient client = HttpUtility.threadSafeClient();
+        HttpClientBuilder httpClientBuilder = new HttpClientBuilder().useSystemProperties();
         if (canInsecureAccess()) {
             try {
-                HttpUtility.setInsecureAccess(client);
+                httpClientBuilder.insecure(true);
```

Comment:
It doesn't look to me that httpClientBuilder.insecure(true) can throw an Exception, is it not just setting boolean variables?